CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.8.4 (2022-02-08)
+--------------------------
+
+- Changes installation to revert ``Yadage`` dependency versions.
```

Comment:
Should we add a new entry for https://github.com/reanahub/reana-commons/commit/4f15ed4857bb57e79b842d0381a073cc05720ce0?